build/metadata.sh

```diff
@@ -9,6 +9,7 @@ OPERATOR_NAME=${OPERATOR_NAME:-service-telemetry-operator}
 IMAGE_BUILDER=${IMAGE_BUILDER:-podman}
 IMAGE_BUILD_ARGS=${IMAGE_BUILD_ARGS:-''}
 IMAGE_TAG=${IMAGE_TAG:-latest}
+# NOTE(vkmc) Where is this env var being used?
```

Comment:
This script is used in generate_bundles.sh

Reply:
A few others as well which aren't used too much any more:

* build/push_container2ocp.sh
* deploy/remove_stf.sh
* deploy/quickstart.sh